tests/migration_test.go

```diff
@@ -2198,7 +2199,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 			It("[test_id:4747] should migrate using for postcopy", func() {
 				vmi := tests.NewRandomFedoraVMIWithGuestAgent()
-				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512Mi")
```

Comment:
This is something we should keep an eye on. I think we have seen issues using less memory for Fedora in the past.

Reply:
libvmi uses 512M, so I think it should be fine?

https://github.com/kubevirt/kubevirt/blob/v1.0.0/tests/libvmi/factory.go#L35

Reply:
Ok, glad to see we don't see problems anymore